open_spiel/games/negotiation.cc

```diff
@@ -617,5 +621,17 @@ std::unique_ptr<State> NegotiationGame::DeserializeState(
   }
 }
 
+std::string NegotiationGame::GetRNGState() const {
```

Comment:
Can you use StrCat here instead, i.e.:

return absl::StrCat(*rng_);

Using std::ostringstream has caused a lot of performance issues in the past. If this isn't possible, then this is fine.

Reply:
I can't seem to find a solution without the stream. Any ideas?

Reply:
Don't worry about it, this is fine. I'll look into it if this turns out to be a bottleneck (which it almost certainly won't be).